common/src/main/java/feast/common/logging/AuditLogger.java

```diff
@@ -61,6 +68,23 @@ public static void logMessage(Level level, MessageAuditLogEntry.Builder entryBui
             .setComponent(buildProperties.getArtifact())
             .setVersion(buildProperties.getVersion())
             .build());
+    Map<String, Object> fluentDLogs = new HashMap<>();
+
+    if (logForwarderProperties.isEnabled()) {
+      FLUENT_LOGGER =
+          FluentLogger.getLogger(
+              "feast",
+              logForwarderProperties.getFluentdHost(),
+              logForwarderProperties.getFluentdPort());
+      fluentDLogs.put("id", entryBuilder.build().getId());
+      fluentDLogs.put("service", entryBuilder.build().getService());
+      fluentDLogs.put("status_code", entryBuilder.build().getStatusCode());
+      fluentDLogs.put("method", entryBuilder.build().getMethod());
+      fluentDLogs.put("request", entryBuilder.build().getRequest());
```

Comment:
Request and response should be in Json, just like the existing audit logger. Otherwise its very hard to parse in BQ.

Reply:
https://github.com/feast-dev/feast/blob/21509be8717674575db6edb6f67378b40043e1e3/common/src/main/java/feast/common/logging/entry/MessageAuditLogEntry.java#L97

Reply:
Updated.